disposable_email_blacklist.conf

```diff
@@ -1544,6 +1560,7 @@ secretemail.de
 secure-mail.biz
 secure-mail.cc
 secured-link.net
+securehost.com.es
```

Comment:
It was proposed by https://mytemp.email/ when you ask to create a new disposable email address